### ISSUE TITLE
The Download link does not work

### ISSUE THREAD

**Issue by Mohamedisaac** [olivierkes/manuskript#1175]
When you click the download link, this error message occurs: 
Fatal error: __autoload() is no longer supported, use spl_autoload_register() instead in /home/clients/90412ec2d8e55d60c960a48d035ee4d5/web/manuskript/wp-includes/compat.php on line 502

**Comment by Latyshevad** [comment]
I confirm. I also tried to go to this URL and did not work. It looks like the site is broken.

**Comment by TheJackiMonster** [comment]
@olivierkes Seems like the webhost updated its PHP version. ^^'

**Comment by respectedriot** [comment]
Yeah, I've also been having this issue. Does anyone have a workaround?

**Comment by TheShadowOfHassen** [comment]
It's worse. Today I checked, and the entire website is down. Manuskript's not The rest of the website. As in https://www.theologeek.ch/ works but not https://www.theologeek.ch/manuskript.

**Comment by andrei-akopian** [comment]
You will download it from the GitHub release page anyway: https://github.com/olivierkes/manuskript/releases

**Comment by TheShadowOfHassen** [comment]
I know. But the entire Manuksript Website is down. GitHub is only useful for the tech-savvy people who already know about Manuksript, we're basically invisible to new users.

We need to get this fixed or set up a new website.

**Comment by TheJackiMonster** [comment]
@TheShadowOfHassen I've already contacted Olivier via mail. Since it's his webserver, he needs to fix it. There's not much else we can do.

I don't think we should set up a new website because we need to adjust all links, potentially twice when it gets fixed. Opening, creating and hosting a website takes time and money. Then search engines will still bring most people to the old website anyway because they know the old link. A new website wouldn't solve that. Because let's be honest: Not-tech-savvy people will go to the website via Google.

Maybe if we have another release and the website is still broken, we can look for a solution. But otherwise binaries are still on Github to download, automated testing still works, changelogs are published on Github as well and packages on Linux distributions as well as flatpak or snap work fine.

So currently this only hurts the announcements of new releases and potential feedback via comments on the site which happens sometimes. Usually I tell most people in the comments to report issues or ask for features here on Github though. So many should already know where to look.

I would think the easiest temporary fix would be to redirect from http://www.theologeek.ch/manuskript to the Github repository page.

**Comment by Mohamedisaac** [comment]
> Yeah, I've also been having this issue. Does anyone have a workaround?

This link from Source Forage works:
https://sourceforge.net/projects/manuskript/

**Comment by andrei-akopian** [comment]
> > Yeah, I've also been having this issue. Does anyone have a workaround?
> 
> This link from Source Forage works: https://sourceforge.net/projects/manuskript/

https://github.com/olivierkes/manuskript/releases

**Comment by TheJackiMonster** [comment]
> > Yeah, I've also been having this issue. Does anyone have a workaround?
> 
> This link from Source Forage works: https://sourceforge.net/projects/manuskript/

This is not an officially supported source. I'd recommend using the releases on Github.

**Comment by TheShadowOfHassen** [comment]
If we're thinking about workaround's endless sky has a website that's linkes to github like endlessky.github.io a website like that would do what we need.

However that would require manuksript to be an organization or something and while I think that would be  a good idea it'd also require Oliver.

Can you set it up to redirect?

**Comment by TheJackiMonster** [comment]
> Can you set it up to redirect?

No. As I said, I don't have administrative access to the webhost. But as mentioned I also doubt a new website from scratch would be a workaround. It's much better to repair the existing website.

**Comment by SConard** [comment]
Just thought I'd update - the website is back up.

**Comment by TheShadowOfHassen** [comment]
Yep it's back.
